bokeh/util/compiler.py

```diff
@@ -160,12 +160,6 @@ class JavaScript(Inline):
     def lang(self):
         return "javascript"
 
-class Eco(Inline):
-
-    @property
-    def lang(self):
-        return "eco"
-
```

Comment:
I very much doubt any users were using this but rates a mention in migration notes